src/library/Collections.nim

```diff
@@ -1760,7 +1760,10 @@ proc defineSymbols*() =
                     elif (hadAttr("lines")):
                         SetInPlace(newStringBlock(InPlaced.s.splitLines()))
                     elif (hadAttr("path")):
-                        SetInPlace(newStringBlock(InPlaced.s.split(DirSep)))
+                        when defined(windows):
+                            SetInPlace(newStringBlock(InPlaced.s.split({'/', '\\'})))
+                        else:
+                            SetInPlace(newStringBlock(InPlaced.s.split(DirSep)))
```

Comment:
```suggestion
                        SetInPlace(newStringBlock(InPlaced.s.split({DirSep, AltSep})))
```

As I have just discovered myself, this is already supported by Nim: https://nim-lang.org/docs/os.html#AltSep

To keep things quite Nim-compliant, I think this could be solved even more easily. 😉

Reply:
Since this is just for the literal, in-place use case of `split.path`, if you could add it as well for the "normal" version (below), it would be great! :)

Reply:
Done!

Reply:
Awesome! :)